example/gluon/mnist/README.md

```diff
@@ -0,0 +1,37 @@
+# MNIST classification example
+
+This script shows a simple example how to do image classification with Gluon. 
+The model is trained on MNIST digits image dataset and the goal is to classify the digits ```0-9```.  The model has the following layout:
+```
+net = nn.Sequential()
+with net.name_scope():
+    net.add(nn.Dense(128, activation='relu'))
+    net.add(nn.Dense(64, activation='relu'))
+    net.add(nn.Dense(10))
+```
+
+The script provides the following commandline arguments: 
+
+
+```
+MXNet Gluon MNIST Example
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --batch-size BATCH_SIZE
+                        batch size for training and testing (default: 100)
+  --epochs EPOCHS       number of epochs to train (default: 10)
+  --lr LR               learning rate (default: 0.1)
+  --momentum MOMENTUM   SGD momentum (default: 0.9)
+  --cuda                Train on GPU with CUDA
+  --log-interval N      how many batches to wait before logging training
+                        status
+```
+
+After one epoch we get the following output vector for the given test image:
+
+<img src="test_image.png" width="250" height="250">
```

Comment:
can this image be put in dmlc/web-data ? Thanks!

Reply:
I moved all images to dmlc/web-data